api/src/main/java/jakarta/servlet/ServletContext.java

```diff
@@ -112,15 +112,15 @@
 
     /**
      * Returns the major version of Jakarta Servlet that this container supports. All implementations that comply with
-     * Version 4.0 must have this method return the integer 4.
+     * Version 5.0 must have this method return the integer 5.
      *
-     * @return 4
+     * @return 5
```

Comment:
Why don't we just remove the hard coded value so we don't forget this again for servlet 6.0?

Reply:
How about this?